### ISSUE TITLE
Rare errors comes from GooglePlayGames.PlayGamesPlatform.GetIdToken

### ISSUE THREAD

**Issue by doterax** [playgameservices/play-games-plugin-for-unity#793]
I analyze unhandled exceptions log from our game, and found this exceptions from your code. I think It might be usefull for you.

Types:
* **java.lang.IllegalStateException: GoogleApiClient must be connected.**
* **java.io.IOException: NetworkError** (when Application.InternetReachability == NotReachable)	

Stack:
```
  at UnityEngine.AndroidJNISafe.CheckException () [0x00000] in :0 
  at UnityEngine.AndroidJNISafe.CallStringMethod (IntPtr obj, IntPtr methodID, UnityEngine.jvalue[] args) [0x00000] in :0 
  at UnityEngine.AndroidJavaObject._Call[String] (System.String methodName, System.Object[] args) [0x00000] in :0 
  at UnityEngine.AndroidJavaObject.Call[String] (System.String methodName, System.Object[] args) [0x00000] in :0 
  at GooglePlayGames.Android.AndroidTokenClient.GetAccountName () [0x00000] in :0 
  at GooglePlayGames.Android.AndroidTokenClient.GetIdToken (System.String serverClientID) [0x00000] in :0 
  at GooglePlayGames.Native.NativeClient.GetIdToken () [0x00000] in :0 
  at GooglePlayGames.PlayGamesPlatform.GetIdToken () [0x00000] in :0 
```
Android versions:
* Android OS 4.4.2 / API-19 (KOT49H/eng.android.1407493065)
* Android OS 4.2.2 / API-17 (JDQ39/vASQ-0)
* Android OS 4.4.4 / API-19 (KTU84P/I9060IXMU0AOG2)
* Android OS 5.0.2 / API-21 (LRX22G/T805XXU1BOJ1)
* Android OS 4.2.2 / API-17 (JDQ39/WW_epad-WW_user_3.1.0.42-user-2
* Android OS 4.3 / API-18 (JSS15J/I9300XXUGOE1)

ProcessorType: ARMv7 VFPv3 NEON
UnityVersion: 5.2.2f1
GPGS Plugin version: 0.9.26
google-play-services.jar version: 27
Affected devices: 700+
Running hours: ~ 20
Unhandled events in your code: 13

**Comment by claywilkinson** [comment]
Thanks for the details!  I think the latest version handles this case now.

**Comment by claywilkinson** [comment]
This is fixed in the 0.9.30 version of the plugin.

**Comment by DengyiLai** [comment]
I hava same problem but my Unity version is only 4.7.2.

I download google sdk from 
https://github.com/playgameservices/play-games-plugin-for-unity/tree/frengle/current-build
It seems no 0.9.30 version. How can I fix this problem?
